docs/blogs/kubecon_cfp.md

```diff
@@ -2,55 +2,49 @@
 
 ## Description
 
-In addition to online services, users have been running distributed AI jobs on
-Kubernetes. Related projects add Kubernetes operators to start distributed deep
-learning programs calling TensorFlow or PyTorch. In these solutions, Kubernetes
-plays the role to launch pods and restart preempted ones. However, such retrials
-often fail due to the same reason that caused the perception -- the lack of
-resources. If couldn't maintain the constant number of worker pods, jobs fail.
-Maintaining the constant number, it becomes gang scheduling. Either case leads
-to low utilization of clusters.
-
-ElasticDL changes the paradox by using the statistical properties of distributed
-learning theory and making jobs tolerable to varying numbers of workers. Taking
-it as the basis, ElasticDL realizes elastic scheduling by introducing a master
-pod per job, to replace the Kubernetes operator per cluster. It makes full use
-of residual resources and improve the utilization significantly.
+In addition to online services, users have been running AI jobs on Kubernetes.
+Related projects add Kubernetes operators to start distributed deep learning
+programs calling TensorFlow or PyTorch. In these solutions, Kubernetes plays the
+role to launch pods and restart preempted ones. However, such retrials often fail
+due to the same reason - the lack of resources. If couldn't maintain the constant
+number of worker pods, jobs fail. Maintaining the constant number, it becomes gang
```

Comment:
`If couldn't maintain the constant number of worker pods, jobs fail.` 
Missing subject. Who couldn't maintain?

Reply:
If the cluster couldn't maintain the constant number of worker pods for the job, the job fails.

Reply:
Update to 
`Jobs would fail if not maintaining the constant number of worker pods.`
We prefer to keep the sentence simple and short to make sure that the section wouldn't exceed the character limit.